react-components/src/queries/__test__/validateQueries.test.js

```diff
@@ -33,7 +33,9 @@ describe.each([['2.4.0', magentoSchema240]])(
 
         it.each(files)('validates the GraphQL request from %s', file => {
             let query = fs.readFileSync(path.join(__dirname, '..', file), 'UTF-8'); // eslint-disable-line
```

Comment:
You should rather simply dynamically `import` each query/file here as any normal exported js module value.

Reply:
tried that. didn't work for some reason.

Reply:
there seems to be a difference between the parsed query and the imported one:
parsed query:
```  
    {
        kind: 'Document',
        definitions: [
          {
            kind: 'OperationDefinition',
            operation: 'query',
            name: [Object],
            variableDefinitions: [],
            directives: [],
            selectionSet: [Object],
            loc: [Object]
          }
        ],
        loc: { start: 0, end: 146 }
     } 
```
imported query : 
```
    {
        default: {
          kind: 'Document',
          definitions: [ [Object] ],
          loc: { start: 0, end: 147 }
        }
    }
```

Reply:
Ignore my comments. I'll update the code.